package.json

```diff
@@ -10,13 +10,19 @@
     "start": "nf start",
     "lint": "eslint --color *.js; exit 0;",
     "test:watch": "mocha -w",
-    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec"
+    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec",
+    "migrate": "node_modules/.bin/sequelize db:migrate",
+    "migrate:undo": "node_modules/.bin/sequelize db:migrate:undo",
+    "migrate:undo:all": "node_modules/.bin/sequelize db:migrate:undo:all",
+    "migrate:test": "node_modules/.bin/sequelize db:migrate --url "
```

Comment:
What does this test?  Can you please include information in an appropriate location that documents thus command?